script.js

```diff
@@ -22,3 +24,248 @@ for (let i = 0; i < menuLinks.length; i += 1) {
+            ever since the 1500s, when an unknown printer took a galley of
+            type and scrambled it 1960s. 
+           </p>
+
+           <p>
+            Lorem Ipsum is simply dummy text of
+            the printing and typesetting industry. Lorem Ipsum has been the
+            industry's standard dummy text ever since the 1500s, when an
+            unknown printer took a galley of type and scrambled it 1960s with
+            the releax map lapora verita.
+           </p>
+         </div>
+         
+         <div class="modal-btns">
+          <div class="see-live">
+            <a href="#">See Live  <img src="/assets/Icon-see live.svg" alt=""></a>
+          </div>
+          <div class="see-source">
+            <a href="#">See Source  <img src="/assets/Vector.png" alt=""></a>
+          </div>
+         </div>
+        </div>
+      </div>
+    </div>
+  </div>
+    `;
+    modalBackground.setAttribute('class', 'modal-background');
+    modalBackground.innerHTML = modalBoxContent;
+    workSection.appendChild(modalBackground);
+    document.body.style.overflowY = 'hidden';
+  }
+
+  if (e.target.classList.contains('btn-bg')) {
+    const modalBackground = document.createElement('div');
+    const modalBoxContent = `
+   <div class="modal-box">
+   <div class="modal-inner">
+     <div class="modal-title" >
+       <div class="modal-header">
+         <h3>Keeping track of hundreds of components website</h3>
+         <div>
+           <img src="./assets/Icon.png" class="close-menu" alt="close-menu" />
+         </div>
+       </div>
+       <br>
+       <ul class="modal-list">
+         <li class="tools-1">HTML</li>
+         <li class="tools-1">Boostrap</li>
+         <li class="tools-1">Ruby on rails</li>
+       </ul>
+     </div>
+     <div class="modal-img-box">
+       <div class="modal-img">
+         <img src="/assets/Snapshoot Portfolio.png" alt="" />
+       </div>
+
+       <div class="modal-details">
+        <div class="modal-info">
+         <p>
+           Lorem Ipsum is simply dummy text of the printing and typesetting
+           industry. Lorem Ipsum has been the industry's standard dummy text
+           ever since the 1500s, when an unknown printer took a galley of
+           type and scrambled it 1960s. 
+          </p>
+
+          <p>
+           Lorem Ipsum is simply dummy text of
+           the printing and typesetting industry. Lorem Ipsum has been the
+           industry's standard dummy text ever since the 1500s, when an
+           unknown printer took a galley of type and scrambled it 1960s with
+           the releax map lapora verita.
+          </p>
+        </div>
+        
+        <div class="modal-btns">
+         <div class="see-live">
+           <a href="#">See Live  <img src="/assets/Icon-see live.svg" alt=""></a>
+         </div>
+         <div class="see-source">
+           <a href="#">See Source  <img src="/assets/Vector.png" alt=""></a>
```

Comment:
:red_circle: Please make sure the right-positioned icons inside these button displays.  As you can none of them is visible 

Your Design
![iconss](https://github.com/Adesina-Abiodun/Microverse-Portfolio-Project/assets/26736582/16d500dc-922b-430b-9379-f5ea63360158)

Figma
![icons](https://github.com/Adesina-Abiodun/Microverse-Portfolio-Project/assets/26736582/a0ee15b7-29d7-4d5f-994e-c118feb93307)